src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/RuleBasedTests.cs

```diff
@@ -526,6 +563,18 @@ internal static class ResourceFilter
                 (string input) => DateTime.TryParse(input, out _),
                 (string input) => bool.TryParse(input, out _),
             };
+
+            public static readonly List<Func<string, bool>> NonPatientTemplates = new List<Func<string, bool>>
+            {
+                // Templates that dosn't contains patient resource
+                _noPatientTemplate.Contains,
+            };
+
+            public static readonly List<Func<string, bool>> MultiplePatientTemplates = new List<Func<string, bool>>
+            {
+                // Templates that contains multiple patient resource
```

Comment:
Nit: typo `contain multiple patient resources`

Reply:
@BoyaWu10 : Fixed the same